source/Halibut/Transport/TcpClientManager.cs

```diff
@@ -77,15 +77,23 @@ public void RemoveClient(TcpClient client)
 
         public void Dispose()
         {
-            var clients = activeClients.ToArray();
-            activeClients.Clear();
+            var clients = activeClients?.ToArray();
```

Comment:
I think this is redundant, as `activeClients` is `readonly`, and is set at the top.

But I'm ok with being safe rather than sorry 🙂